### DISCUSSION TITLE
Version 8 UI notes

### DISCUSSION THREAD

**Discussion by mouse-the** [Tribler/tribler#8347] [General]
Hi everybody. I was using the Flathub version of Tribler, which has made me a bit late to version 8. (Flatpaks do not seem to be a good solution for me, and I am now back on Tribler's github release.)

After using the web UI for a few hours I have some notes. These could probably turn into feature requests or bug reports, but I am not sure anyone else cares about this stuff, so I will see if anyone wants to discuss it. 
I am sad to see the desktop UI go, but I understand.

Sorting Downloads
I used to sort first by date added, then by progress, then by down speed. This made it easy to find downloads which had completed since the last time I checked the interface or see which downloads have been stalled for a long period of time. The web gui seems to arbitrarily order downloads that have matching fields in the sorted column. 

Keyboard Navigation
I used to scroll through downloads with the arrow keys. For instance this was an easy way to make sure all the completed downloads had been moved out of the initial download directory.

(Between those 2 I have no way of finding recently completed downloads that need my attention, they are just lost among the seeding downloads. (Is seeding working?) (Yes, I am a bit messy with a 3 digit number of downloads.))

Move file storage
Is not accessible through context-click, not easily accessible on the (almost empty) toolbar, lacks history, and uses a very minimal  directory chooser. (It does not even sorting by anything I can figure out).

There are no tool tips for the icon-only toolbar items. (Combined with no context menu...)

When scrolling the downloads list the Status field renders on top of the header bar.

Add torrent:Import torrent from magnet/URL
No longer accepts a raw torrent infohash (need to manually prepend "magnet:?xt=urn:btih:" )

There is now no obvious way to deselect some files in a download before starting it. (Also, deselecting almost all files seems to be even more difficult than it was before.)
Edited to add: Unless the first checkbox under settings is obvious. Ooops.

I do miss the ETA display, but that is minor.

The progress display can now round up to 100%, which seems wrong to me, also minor.

**Comment by qstokkink** [comment]
Thanks for the feedback: we definitely care! We'll have a look at how much of this we can implement for the 8.1 release (next year).

**Comment by mouse-the** [comment]
Thanks for your work, and have a good break!

I found the setting "Always ask download settings" which lets me deselect files before starting a download. So that is ok.

**Comment by kitsorfan** [comment]
I also think we need to look forward restoring the desktop UI.
